generators/client/templates/angular/package.json.ejs

```diff
@@ -78,6 +78,7 @@
     "html-loader": "0.5.0",
     "html-webpack-plugin": "3.0.0",
     "husky": "0.14.3",
+    "istanbul-instrumenter-loader": "3.0.0",
```

Comment:
`3.0.0` doesn't support webpack 4 right ?

Reply:
I didn't have issue about this...